sources/assets/exegol/load_supported_setups.sh

```diff
@@ -52,6 +52,14 @@ function deploy_vim() {
   fi
 }
 
+function deploy_nvim () {
+  #### neovim deployment
+  if [ -d "$MY_Setup_PATH/nvim" ]; then
```

Comment:
If the directory doesn't exist, you should create it.
If possible / applicable, you can also create a template file (not mandatory at all, just for a better user experience maybe)